modules/SofaConstraint/SlidingConstraint.h

```diff
@@ -66,6 +67,7 @@ class SlidingConstraint : public core::behavior::PairInteractionConstraint<DataT
     Data<int> m1; ///< index of the spliding point on the first model
     Data<int> m2a; ///< index of one end of the sliding axis
     Data<int> m2b; ///< index of the other end of the sliding axis
+    Data<Deriv> mforce; ///< interaction force
```

Comment:
Please use `d_` prefix for Data members even if it's not (yet) consistent with existing Data members.
See https://github.com/sofa-framework/sofa/blob/master/GUIDELINES.md#naming